feeds/grouping-and-source-pinning.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: Grouping and Source Pinning
+parent: Feeds
+nav_order: 1
+---
+
+# Grouping and Source Pinning
+
+The default NuGet behavior with feeds declared in NuGet.config is to query each feed for every package. For large repositories, this can lead to performance and network bottlenecks.
+
+By specifying a group for a feed, only the direct and transitive dependencies of a package defined with a `<PackageVersion>` in the same group will use this feed. More information about adding a `CentralManagementGroup` to a `<PackageVersion>` can be found in the [documentation for package version grouping](http://localhost:4000/package-versions/grouping-and-overrides/grouping-and-overrides/).
+
+## Global Group
+
+A global group will be automatically created that includes all `<PackageSource>`, `<PackageVersion>`, and `<PackageReference>` that do not have a `CentralManagementGroup` specified.
+
+`<PackageSource>` that belong to the global group will also be queried for `<PackageReference>` that belong to a `CentralManagementGroup` unless specified by [Feed Scoping](/feeds/grouping-and-source-pinning/#feed-scoping).
```

Comment:
Nit: Can we have a simpler name for `CentralManagementGroup`?

Reply:
I agree that it should be shorter, but I just don't have a good answer for what it _should_ be.

I was just discussing this with @loic-sharma, and we walked through the process that I used to arrive at the current name.

If we use the simple term `Group`, that overloads a term already used in MSBuild. The user would put a `Group` in an `ItemGroup`.

If we use `PackageGroup`, then you are adding a `PackageGroup` to `<PackageVersion>`, `<PackageReference>`, and `<PackageSource>`.

I agree that it would be great if we had a short, descriptive term, but I haven't figured it out yet.
- Shorter than `CentralManagementGroup`
- Easy to search for on Bing/Google with the terms NuGet and MSBuild
- Doesn't mix concepts already in NuGet and MSBuild

Reply:
Other suggestions that I believe aren't used in MSBuild today: `Scope`, `Area`.

Reply:
> If we use `PackageGroup`, then you are adding a `PackageGroup` to `<PackageVersion>`, `<PackageReference>`, and `<PackageSource>`.

TBH, I really like `PackageGroup` in combination with the other `Package*`-names. It's unique and concise and there's no need to invent new terms like "central" & "management". It's just an additional layer to manage `packages`. 

Sure, this whole topic is about "centrally managing packages" but I don't think these terms should be reflected in code. Who knows, maybe one day `Directory.packages.props` might even be the default for new project/solution templates. If someone decides to use Directory.Packages.props, s/he should will just be introduced to two simple new names: `PackageVersion` and `PackageGroup`.

Disclaimer: I'm not a native english speaker, so this just might sound good to me but not to native english speakers?!